conda_forge_tick/utils.py

```diff
@@ -524,9 +509,7 @@ def parse_recipe_yaml_local(
     for_pinning : bool, optional
         If True, render the recipe.yaml for pinning migrators, by default False.
     platform : str, optional
-        The platform (e.g., 'linux', 'osx', 'win').
-    arch : str, optional
-        The CPU architecture (e.g., '64', 'aarch64').
+        The platform (e.g., 'linux-64', 'osx-arm64', 'win-64').
```

Comment:
See my comment on platform vs arch below. We should call this one `platform_arch` if this is what we intend or similar.